src/System.Windows.Forms/src/System/Windows/Forms/UpDownBase.UpDownButtons.UpDownButtonsAccessibleObject.DirectionButtonAccessibleObject.cs

```diff
@@ -44,7 +44,7 @@ public override Rectangle Bounds
                             }
 
                             // Convert to screen coords
-                            return (((UpDownButtons)_parent.Owner).ParentInternal).RectangleToScreen(bounds);
+                            return ((UpDownButtons?)_parent.Owner)?.ParentInternal?.RectangleToScreen(bounds) ?? Rectangle.Empty;
```

Comment:
`_parent.Owner` can't be null here, we'd have failed on line:32 with a NRE.
```suggestion
                            return ((UpDownButtons)_parent.Owner).ParentInternal?.RectangleToScreen(bounds) ?? Rectangle.Empty;
```